src/test/java/com/hillel/items_exchange/util/ChildDtoCreatingUtil.java

```diff
@@ -0,0 +1,48 @@
+package com.hillel.items_exchange.util;
+
+import com.hillel.items_exchange.dto.ChildDto;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static com.hillel.items_exchange.util.JsonConverter.asJsonString;
+
+public class ChildDtoCreatingUtil {
+    public static String getValidChildDtoForCreate() {
+        ChildDto dto = new ChildDto(0, "male", LocalDate.of(2019, 3, 3));
```

Comment:
I see duplication of DTO here - create them as the class' variables and pass them into methods.